### PR TITLE
Fixed components export + MultiSelect visibility issue

### PR DESCRIPTION
**Pull request by piny4man**

### CODE REVIEW

File: src/components/MultiSelect/styles.ts [hunk 1]
```diff
@@ -20,24 +16,29 @@ export default styled.article`
...
   background: ${ ({ theme }) => theme.colors.grey.light };
+  cursor: pointer;
 
   &.opened {
     border-bottom-color: ${ ({ theme }) => theme.colors.grey.medium };
     box-shadow: 0 -2px 6px 0 ${ ({ theme }) => theme.colors.typography.medium }1F;
     background: white;
 
+    .down {
+      transform: rotate(180deg);
```
Comment (mfmontenegro) [inline src/components/MultiSelect/styles.ts]: change to icon up instead of rotating it

File: src/components/MultiSelect/styles.ts [hunk 2]
```diff
@@ -99,6 +109,15 @@ export default styled.article`
     }
   }
 
+  .down {
+    position: absolute;
```
Comment (mfmontenegro) [inline src/components/MultiSelect/styles.ts]: Make it relative. Set space-between to parent or make it margin-left auto

File: src/components/MultiSelect/index.tsx [hunk 3]
```diff
@@ -40,6 +41,8 @@ const MultiSelect: React.FC<IProps> = props => {
     if (isOpen && !!openRef && event.target !== openRef) setIsOpen(false)
   }
 
+  const dropdownIcon = isOpen ? 'up' : 'down'
```
Comment (mfmontenegro) [inline src/components/MultiSelect/index.tsx]: just 'icon'. It's a multi select but you call it dropdown which is another different term.

File: src/components/MultiSelect/index.tsx [hunk 4]
```diff
@@ -58,6 +61,7 @@ const MultiSelect: React.FC<IProps> = props => {
       <div className="selected" ref={ divRef } data-text={ placeholder } onClick={ handleOpen }>
         { selectedLabel }
       </div>
+      <Icon className="dropdown-icon" icon={ dropdownIcon }/>
```
Comment (mfmontenegro) [inline src/components/MultiSelect/index.tsx]: just className="icon"

File: src/components/MultiSelect/styles.ts (hunk 2 above)
Comment (alberss) [reply src/components/MultiSelect/styles.ts]: Done

File: src/components/MultiSelect/styles.ts (hunk 1 above)
Comment (alberss) [reply src/components/MultiSelect/styles.ts]: Done

File: src/components/MultiSelect/index.tsx (hunk 3 above)
Comment (alberss) [reply src/components/MultiSelect/index.tsx]: Done

File: src/components/MultiSelect/index.tsx (hunk 4 above)
Comment (alberss) [reply src/components/MultiSelect/index.tsx]: Done

**Pull request merged**